### PR TITLE
bb.org:configs: add latest libiio package

### PR DESCRIPTION
**Pull request by Pillar1989**
There is no python wrapper version in the original debian source, so we can add it here after recompiling.
```
debian@beaglebone:/var/lib/cloud9/libiio$ dpkg --contents  /var/lib/cloud9/libiio/build/libiio-0.18.g3c7d514-Linux.deb
drwxr-xr-x root/root         0 2020-01-30 08:50 ./etc/
drwxr-xr-x root/root         0 2020-01-30 08:50 ./etc/avahi/
drwxr-xr-x root/root         0 2020-01-30 08:50 ./etc/avahi/services/
-rw-r--r-- root/root      1074 2020-01-30 08:12 ./etc/avahi/services/iio.service
drwxr-xr-x root/root         0 2020-01-30 08:50 ./lib/
drwxr-xr-x root/root         0 2020-01-30 08:50 ./lib/udev/
drwxr-xr-x root/root         0 2020-01-30 08:50 ./lib/udev/rules.d/
-rw-r--r-- root/root       121 2020-01-30 08:49 ./lib/udev/rules.d/90-libiio.rules
drwxr-xr-x root/root         0 2020-01-30 08:50 ./usr/
drwxr-xr-x root/root         0 2020-01-30 08:50 ./usr/bin/
-rwxr-xr-x root/root     27648 2020-01-30 08:50 ./usr/bin/iio_adi_xflow_check
-rwxr-xr-x root/root     46724 2020-01-30 08:49 ./usr/bin/iio_attr
-rwxr-xr-x root/root     18664 2020-01-30 08:49 ./usr/bin/iio_genxml
-rwxr-xr-x root/root     33464 2020-01-30 08:49 ./usr/bin/iio_info
-rwxr-xr-x root/root     32152 2020-01-30 08:49 ./usr/bin/iio_readdev
-rwxr-xr-x root/root     14008 2020-01-30 08:50 ./usr/bin/iio_reg
-rwxr-xr-x root/root     32304 2020-01-30 08:50 ./usr/bin/iio_writedev
drwxr-xr-x root/root         0 2020-01-30 08:50 ./usr/include/
-rw-r--r-- root/root     72221 2020-01-30 08:12 ./usr/include/iio.h
drwxr-xr-x root/root         0 2020-01-30 08:50 ./usr/lib/
drwxr-xr-x root/root         0 2020-01-30 08:50 ./usr/lib/arm-linux-gnueabihf/
lrwxrwxrwx root/root         0 2020-01-30 08:50 ./usr/lib/arm-linux-gnueabihf/libiio.so -> libiio.so.0
lrwxrwxrwx root/root         0 2020-01-30 08:50 ./usr/lib/arm-linux-gnueabihf/libiio.so.0 -> libiio.so.0.18
-rw-r--r-- root/root    365596 2020-01-30 08:49 ./usr/lib/arm-linux-gnueabihf/libiio.so.0.18
drwxr-xr-x root/root         0 2020-01-30 08:50 ./usr/lib/arm-linux-gnueabihf/pkgconfig/
-rw-r--r-- root/root       240 2020-01-30 08:48 ./usr/lib/arm-linux-gnueabihf/pkgconfig/libiio.pc
drwxr-xr-x root/root         0 2020-01-30 08:50 ./usr/lib/python3.5/
drwxr-xr-x root/root         0 2020-01-30 08:50 ./usr/lib/python3.5/site-packages/
drwxr-xr-x root/root         0 2020-01-30 08:50 ./usr/lib/python3.5/site-packages/__pycache__/
-rw-r--r-- root/root     30947 2020-01-30 08:50 ./usr/lib/python3.5/site-packages/__pycache__/iio.cpython-35.pyc
-rw-r--r-- root/root     28109 2020-01-30 08:48 ./usr/lib/python3.5/site-packages/iio.py
-rw-r--r-- root/root       247 2020-01-30 08:50 ./usr/lib/python3.5/site-packages/libiio-0.18-py3.5.egg-info
drwxr-xr-x root/root         0 2020-01-30 08:50 ./usr/sbin/
-rwxr-xr-x root/root    184148 2020-01-30 08:50 ./usr/sbin/iiod
```

### CODE REVIEW

**Comment by RobertCNelson** [comment]
@Pillar1989 & @jadonk , i should have asked this question, earlier... Why are we using stretch for this kit?  This pull request won't work, python3-evdev doesn't exist in stretch, and libiio already has python3 libraries in debian buster libiio. (i had previously tried to backport buster's libiio to stretch, and a python3-evdev was painful when i started it last week.)

https://packages.debian.org/source/buster/libiio

https://packages.debian.org/buster/python3-evdev

I feel like we should switch to buster for this kit...

Regards,

File: configs/bb.org-debian-stretch-iot-grove-kit-v4.14.conf [hunk 1]
```diff
@@ -116,6 +116,7 @@ deb_additional_pkgs="	\
 	python3-pyaudio \
 	python3-tqdm \
 	portaudio19-dev \
+	python3-evdev \
```
Comment (RobertCNelson) [inline configs/bb.org-debian-stretch-iot-grove-kit-v4.14.conf]: This package doesn't exist in Debian Stretch: https://packages.debian.org/buster/python3-evdev

File: configs/bb.org-debian-stretch-iot-grove-kit-v4.14.conf [hunk 2]
```diff
@@ -148,6 +148,7 @@ repo_rcnee_pkg_list="	\
 	firmware-am57xx-opencl-monitor	\
 	gpiod	\
 	linux-image-4.14.108-ti-r127	\
+	seeed-modules-4.14.108-ti-r127	\
```
Comment (RobertCNelson) [inline configs/bb.org-debian-stretch-iot-grove-kit-v4.14.conf]: This is package is already installed:

https://github.com/beagleboard/image-builder/blob/master/scripts/chroot.sh#L730-L746

Regards,

Comment (Pillar1989) [reply configs/bb.org-debian-stretch-iot-grove-kit-v4.14.conf]: Strangely, we did not find it in the Image.
 https://rcn-ee.net/rootfs/bb.org/testing/2020-01-23/stretch-iot-music-kit/

**Comment by Pillar1989** [comment]
> @Pillar1989 & @jadonk , i should have asked this question, earlier... Why are we using stretch for this kit? This pull request won't work, python3-evdev doesn't exist in stretch, and libiio already has python3 libraries in debian buster libiio. (i had previously tried to backport buster's libiio to stretch, and a python3-evdev was painful when i started it last week.)
> 
> https://packages.debian.org/source/buster/libiio
> 
> https://packages.debian.org/buster/python3-evdev
> 
> I feel like we should switch to buster for this kit...
> 
> Regards,

@jadonk I also want to know,  the stretch more mature and stable? Or something else?

**Comment by RobertCNelson** [comment]
and i just added a buster version:

https://github.com/beagleboard/image-builder/commit/30234b01cd17778cddeb48523027c9a06619eb9f

Regards,

File: configs/bb.org-debian-stretch-iot-grove-kit-v4.14.conf (hunk 2 above)
Comment (RobertCNelson) [reply configs/bb.org-debian-stretch-iot-grove-kit-v4.14.conf]: I just looked at the build log:

```E: Invalid operation seeed-modules-4.14.108-ti-r126
Reading package lists...
Building dependency tree...
Reading state information...
The following NEW packages will be installed:
  ti-cmem-4.15.00.02-modules-4.14.108-ti-r126
```
humm...

I'm an idiot sometimes.. (it helps if apt-get has the "install" keyword on it..)

https://github.com/beagleboard/image-builder/commit/954106b65ee786f4e7d3a9605f41deea458e6620

Regards,

**Comment by RobertCNelson** [comment]
Test images with today's fixes are being uploaded to here:

https://rcn-ee.net/rootfs/bb.org/testing/2020-01-30/

Regards,

**Comment by jadonk** [comment]
@Pillar1989 is this working now?

**Comment by Pillar1989** [comment]
@jadonk  works now

**Pull request closed without merging**